### ISSUE TITLE
Unrecoverable Error

### ISSUE THREAD

**Issue by RamaParedes** [Soreepeong/XivAlexander#549]
I'm having this weird error when I'm trying to open the game with Alexander, I don't know if anyone is having the same error or what
![Sin título](https://user-images.githubusercontent.com/52410319/211946973-8b7a3dd8-2c9d-45bf-a323-fe400729f688.png)

**Comment by PZDuck** [comment]
> I don't know if anyone is having the same error or what

Please check previous issues

**Comment by exaflareenjoyer** [comment]
i am experiencing the same error too. other issues seems to be the same as well. there is an error right now, so we're all waiting for updates

**Comment by Pox4ever** [comment]
> i am experiencing the same error too. other issues seems to be the same as well. there is an error right now, so we're all waiting for updates

It's not guaranteed that he is working on an update and it's not even written in docs or anywhere else his support date for this tool, it's not his work to do it though, it's free. So don't put too much faith in it. I would suggest using NoClippy since it is already working since yesterday, and hope for this one to be updated.

**Comment by stealyourwife** [comment]
how are you able to access NoClippy while Dalamud's not working?

**Comment by Pox4ever** [comment]
> how are you able to access NoClippy while Dalamud's not working?

On their discord channel, you can get the beta key for testing purposes and it works.

**Comment by stealyourwife** [comment]
on their discord, where am I able to get the beta key for it? apparently, distribution for testing is currently unavailable, so I'm wondering if I'm missing this beta key somewhere

**Comment by Pox4ever** [comment]
> on their discord, where am I able to get the beta key for it? apparently, distribution for testing is currently unavailable, so I'm wondering if I'm missing this beta key somewhere

Yeah, I think they've closed it already because another FFXIV patch will come in tonight for a maintenance, so they won't be releasing an official public version today.

**Comment by stealyourwife** [comment]
I appreciate the responses, hope they'll have it back soon since I'm in desperate need of NoClippy due to TEA reclears

**Comment by Pox4ever** [comment]
> I appreciate the responses, hope they'll have it back soon since I'm in desperate need of NoClippy due to TEA reclears

If you have twitter feel free to reach me out through DM. It's on my git profile.

**Comment by stealyourwife** [comment]
I sent you a follow, though I'm not able to send a DM to you specifically for some reason

**Comment by Pox4ever** [comment]
> I sent you a follow, though I'm not able to send a DM to you specifically for some reason

Strange, I don't see your follow either, lol

Send me a msg on discord then
Pox4eveR#6666

**Comment by csianne** [comment]
> > i am experiencing the same error too. other issues seems to be the same as well. there is an error right now, so we're all waiting for updates
> 
> It's not guaranteed that he is working on an update and it's not even written in docs or anywhere else his support date for this tool, it's not his work to do it though, it's free. So don't put too much faith in it. I would suggest using NoClippy since it is already working since yesterday, and hope for this one to be updated.

I am testing dalamud since yday and NoClippy was never updated? I double checked the releases on their discord and its not here either?

**Comment by Pox4ever** [comment]
> > > i am experiencing the same error too. other issues seems to be the same as well. there is an error right now, so we're all waiting for updates
> > 
> > 
> > It's not guaranteed that he is working on an update and it's not even written in docs or anywhere else his support date for this tool, it's not his work to do it though, it's free. So don't put too much faith in it. I would suggest using NoClippy since it is already working since yesterday, and hope for this one to be updated.
> 
> I am testing dalamud since yday and NoClippy was never updated? I double checked the releases on their discord and its not there either?

You prolly on the unofficial one. It is there. UnknownX7 is the owner of the repo.

**Comment by csianne** [comment]
Ah right I see the repo is updated, but they didnt releases officialy yet

**Comment by CSIXIV** [comment]
> > > > i am experiencing the same error too. other issues seems to be the same as well. there is an error right now, so we're all waiting for updates
> > > 
> > > 
> > > It's not guaranteed that he is working on an update and it's not even written in docs or anywhere else his support date for this tool, it's not his work to do it though, it's free. So don't put too much faith in it. I would suggest using NoClippy since it is already working since yesterday, and hope for this one to be updated.
> > 
> > 
> > I am testing dalamud since yday and NoClippy was never updated? I double checked the releases on their discord and its not there either?
> 
> You prolly on the unofficial one. It is there. UnknownX7 is the owner of the repo.

I just realized I have access to the beta of Dalamud but I see NoClippy being off too. I see the repo is updated tho, how do I import it manually into dalamud?

**Comment by Pox4ever** [comment]
> Ah right I see the repo is updated, but they didnt released officialy yet

I feel you bro, people saying that the owner didn't update it for months but his last commit is from like 3 weeks ago. I wish they at least learned/googled how to read git repos.
lmfao

I just gave up...

**Comment by Swooshyy** [comment]
Having this exact same issue aswell. Downloaded today. Gotta have to wait for a fix.

**Comment by csianne** [comment]
> Having this exact same issue aswell. Downloaded today. Gotta have to wait for a fix.

this is fixed since yesterday, see https://github.com/Soreepeong/XivAlexander/commit/06f8511b8212699e3347ccfb0a97fad4aa30b8e4
and
https://github.com/Soreepeong/XivAlexander/releases/tag/v1.14.8.55

**Comment by Swooshyy** [comment]
> > Having this exact same issue aswell. Downloaded today. Gotta have to wait for a fix.
> 
> this is fixed since yesterday, see [06f8511](https://github.com/Soreepeong/XivAlexander/commit/06f8511b8212699e3347ccfb0a97fad4aa30b8e4) and https://github.com/Soreepeong/XivAlexander/releases/tag/v1.14.8.55

Thanks a bunch! will look into it once i am home.
